applications/app_balance.c

```diff
@@ -419,13 +423,18 @@ static void calculate_setpoint_interpolated(void){
 }
 
 static void apply_constanttilt(void){
-	// Nose angle adjustment
+	// Nose angle adjustment, add variable then constant tiltback
+	float constanttilt_target = 0;
+	if (fabsf(erpm) > tiltback_variable_max_erpm) {
+		constanttilt_target = tiltback_variable * tiltback_variable_max_erpm * SIGN(erpm);
```

Comment:
This line can be simplified to 
`constanttilt_target = balance_conf.tiltback_variable_max * SIGN(erpm);`

Reply:
Good suggestion! But remember what we did with the sign earlier? So we'd need to do fabsf of that, right?